File: front-mecanopecas/src/dtos/response/orcamentoReportResponseDTO.ts
import { OrcamentoResponseDTO } from './orcamentoResponseDTO';

export interface OrcamentoReportResponseDTO {
    titulo: string;
    solicitante: string;
    dataGeracao: Date;
    dataInicio: Date;
    dataFim: Date;
    dados: OrcamentoResponseDTO[];
}

File: back-mecanopecas/src/main/java/br/com/mecanopecas/mecanopecas/util/dtos/response/OrcamentoReportResponseDTO.java
package br.com.mecanopecas.mecanopecas.util.dtos.response;

import java.time.LocalDate;
import java.time.LocalDateTime;
import java.util.List;

public record OrcamentoReportResponseDTO(
        String titulo,
        String solicitante,
        LocalDateTime dataGeracao,
        LocalDate dataInicio,
        LocalDate dataFim,
        List<OrcamentoResponseDTO> dados
) {}
